### PR TITLE
Add Qualys scanner

### PR DESCRIPTION
**Pull request by mathuriga**
Start a scan.
Cancel a scan.

## Security checks
 - Followed secure coding standards in http://wso2.com/technical-reports/wso2-secure-engineering-guidelines? yes
 - Ran FindSecurityBugs plugin and verified report? yes
 - Confirmed that this PR doesn't commit any keys, passwords, tokens, usernames, or other secrets? yes

### CODE REVIEW

File: internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/handler/QualysApiInvoker.java [hunk 1]
```diff
@@ -0,0 +1,230 @@
...
+    private String createReportEndpoint;
+    private String downloadReportEndpoint;
+    private String getReportStatusEndpoint;
+
+    public QualysApiInvoker() {
+
+        // Platform URL of Qualys scanner.
+        String host = QualysScannerConfiguration.getInstance().getHost();
+
+        // Retry time interval
```
Comment (ayomawdb) [inline internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/handler/QualysApiInvoker.java]:
```suggestion
        // Retry time interval.
```

File: internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/handler/StatusHandler.java [hunk 2]
```diff
@@ -0,0 +1,316 @@
...
+    private final class CheckStatusTask implements Runnable {
+
+        @Override public void run() {
+            if (log.isDebugEnabled()) {
+                String message = " Scheduled Executor Service is started to check scan status periodically";
+                log.debug(new CallbackLog(scanContext.getJobID(), message));
+            }
+            String status;
+            try {
+
```
Comment (ayomawdb) [inline internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/handler/StatusHandler.java]:
```suggestion
```

File: internal/scan-manager/common/common/src/main/java/org/wso2/security/tools/scanmanager/common/util/HTTPUtil.java [hunk 3]
```diff
@@ -64,10 +85,121 @@
      * @return a response entity object
      * @throws RestClientException when an error occurs while initiating the request
      */
-    public static ResponseEntity<String> sendDelete(HTTPRequest httpRequest) throws RestClientException {
+    public static ResponseEntity<String> sendDELETE(HTTPRequest httpRequest) throws RestClientException {
```
Comment (ayomawdb) [inline internal/scan-manager/common/common/src/main/java/org/wso2/security/tools/scanmanager/common/util/HTTPUtil.java]: IMO, `sendDelete` is the correct method name here. Please create a issue to correct this commonly across scanner in future.  (for other methods too)

Comment (mathuriga) [reply internal/scan-manager/common/common/src/main/java/org/wso2/security/tools/scanmanager/common/util/HTTPUtil.java]: Created issue #131 to track this.

File: internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/handler/StatusHandler.java (hunk 2 above)
Comment (mathuriga) [reply internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/handler/StatusHandler.java]: Resolved

File: internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/handler/QualysApiInvoker.java (hunk 1 above)
Comment (mathuriga) [reply internal/scan-manager/scanners/qualys/src/main/java/org/wso2/security/tools/scanmanager/scanners/qualys/handler/QualysApiInvoker.java]: Resolved

**Pull request merged**